articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
-If you change the `Sharing setting` on a folder all content within that folder will inherit that change. Just as your are able to control who can access your project, you can also restrict or open portions of your project using the Sharing setting for a folder or subfolder. To set up local Sharing settings select the Share icon from the chosen Project, Folder, File or Table.
-Inheritance is illustrated in the example below. In the diagram, the projects A and B carry the “share broadly” (public) `Sharing setting` designated by the globe icon. No local sharing settings are shown for folders 1 and 2 and Table 3 in Project A. Since these folders and table do not have a Sharing setting specified, they inherit the `sharing setting` of the project: “share broadly.” By contrast, in Project B the folder 4 and Table 6 have a restricted `Sharing setting`, designated with the lock icon. File 5 has no local sharing setting and therefore it inherits the `sharing setting` of the project: “share broadly.” It is important to note that if you were to move File 5into the restricted Folder 4 then File 5’s Sharing setting will be reset to the restricted Sharing settingof Folder 4: “share sparsely”.
+### Sharing content within a Project 
+You can also adjust the `Sharing setting` for `Folders`, `Files`, and `Tables`. By default, all of the content residing within a `Folder` or `Project` inherits the `Sharing setting` of its parent `Folder` or `Project`. To create a local `Sharing setting` specific only to a particular `Folder`, `File` or `Table`, open the `Folder`, `File` or `Table` and click on its `Share` button and adjust the `Sharing setting` just like you would for a `Project`. It is important to note that you **cannot** set local sharing settings for `Wikis` and `Discussion Forums`. `Wikis` and `Discussion Forums` inherit the `Sharing setting` of their parent `Project`.
+
+Local `Sharing settings` are illustrated in the diagram below. `Projects` A and B have a **Public** `Sharing setting` (notice the globe icon on the `Share` button). `Folders` 1 and 2 and `Table` 3 in `Project` A _do not_ have a local `Sharing setting` specified. Because they do not have a local `Sharing setting` specified, they inherit the **Public** `Sharing setting` of `Project` A. By contrast, in `Project` B the `Folder` 4 and `Table` 6 _do_ have local `Sharing settings`. Unlike their parent `Project` B, `Folder` 4 and `Table` 6 have the **Private** `Sharing setting` (notice the lock icon on the `Share` buttons). However, `File` 5 has no local `Sharing setting`; it inherits the **Public** `Sharing setting` of `Project` B. But, if you move `File` 5 into `Folder` 4, then its **Public** `Sharing setting` will be reset to the `Sharing setting` of `Folder` 4: **Private**.
 
```

Comment:
It does not seem necessary to indicate that the files and folders refers to the synapse objects in this case?  It makes the highligthing excessive making it less likely to be able to draw the users attention to the truly important content.